server.js

```diff
@@ -346,54 +344,6 @@ async function generateDevelopmentBuild() {
     next();
   });
 
-  //eslint-disable-next-line no-unused-vars
-  app.get("/proxy{/*remote}", function (req, res, next) {
```

Comment:
The `app.param("remote"...` block above is now unused (the param is the `remote` portion of the url specified here).